cds-snc/cppd-medical-report
# Fix node version at 12.13.1

# Description

This is an attempt to see if fixing the Node version fixes some bugs being experienced uniquely on Heroku deplos.

# Test Instructions

1. Go to /en/add_conditions
1. See if the "Upload" button is styled
